docs/contribute/saros-testing-framework.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Saros Testing Framework (STF)
+---
+
+# {{ page.title }}
+{:.no_toc}
+
+{% include toc.html %}
+
+For further information
+about STF take a look at the [STF User
+Manual](http://saros-build.imp.fu-berlin.de/stf/STF_Manual.pdf).
+
+## Eclipse
+
+### Prerequisites
+
+* Development environment with **Eclipse 4.6**. It is also possible to use newer version, but some buttons are renamed in newer version which leads to a few error.
```

Comment:
Developers should not use other versions. Imagine you write an STF test that works fine in Eclipse 4.8 but always fail in Eclipse 4.6

Reply:
Done